app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new
+  end
```

Comment:
If you don't want to add anything in your controller, you can just render the view directly from the router